birdvoxdetect/core.py

```diff
@@ -442,15 +438,20 @@ def process_file(
 
         # Classify species.
         rows = []
-        none_peak_ids = []
         for peak_id, th_peak_loc in enumerate(th_peak_locs):
-            row, json_dict = classify_species(
+            consistent_pred_dict, json_dict = classify_species(
                 classifier, chunk_pcen, th_peak_loc, taxonomy
             )
-            if row is None:
-                none_peak_ids.append(peak_id)
-                continue
-            rows.append(row)
+            rows.append(
+                {
+                    "Order": consistent_pred["coarse"]["common_name"],
```

Comment:
I believe `consistent_pred` should be `consistent_pred_dict`

Reply:
thanks, fixed